spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -634,11 +629,17 @@ string or a `Claims` enum value. The string form would allow for specifying non-
 claims while the `Claims` enum approach guards against typos.
 
 #### Handling of Non-RequestScoped Injection of Claim Values
+
+MP-JWT implementations are required to support a claim value injection into an @ApplicationScoped scope. A warning may be issued when the @ApplicationScoped injection point is a primitive or wrapper type.
```

Comment:
I would probably rewrite this a little bit to:

MP-JWT implementations are required to support a claim value injection into  **@ApplicationScoped scoped beans**. A warning may be issued when the injection point is a **primitive type (or equivalent wrapper)**.

Reply:
@radcortez What about `String` ?

Reply:
In practice byte[] is a String and the primitive type :D

Is was just to clarify that the wrappers refer to the wrapper of the primitive type and not just any wrapper. The original text might give the impression that any wrapper type is supported.

Reply:
Perhaps it is simpler `A warning may be issued when the injection point is not a 'ClaimValue' or 'Provider' interface`.

Reply:
Sounds good.